src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -388,6 +388,10 @@ public void rollbackTransaction(){
         private static final String MQTT_PERSISTENCE_FALSE          = "false";
         private static final String MQTT_PERSISTENCE_PROPERTY       = "ch.hevs.cloudio.endpoint.persistence";
         private static final String MQTT_PERSISTENCE_DEFAULT        = MQTT_PERSISTENCE_TRUE;
+        private static final String MQTT_UPDATE_LIMIT_PROPERTY      = "ch.hevs.cloudio.endpoint.update-persistence-limit";
```

Comment:
`ch.hevs.cloudio.endpoint.default-persistence.update-limit` as we will soon support custom persistence layer